lib/client/overlays/progress.js

```diff
@@ -112,7 +114,10 @@ const update = (percent) => {
 
 const reset = (svg) => {
   svg.classList.add(`${ns}-hidden`);
-  setTimeout(() => update(0), 1e3);
+  resetTimer = setTimeout(() => {
+    resetTimer = null;
+    update(0);
+  }, 3e3);
```

Comment:
How / why did you arrive at choosing 3 seconds instead of 1 second?

Reply:
It should be 3e2, the same as CSS transition duration.